src/Controller/contact.php

```diff
@@ -7,11 +7,11 @@
     Nom : " . $_POST["lname"] . "
     Prenom : " . $_POST["fname"] . "
     Email : " . $_POST["email"] . "
-    Sujet : " . $_POST["sujet"] . "
-    Message :" . $_POST["subject"];
+    Sujet : " . $_POST["topic"] . "
```

Comment:
controller must have conventionnal naming with Controller prefix
This file should be a class and render a file or redirect to another one